web/blog/2023-11-21-guide-to-wsl.md

```diff
@@ -0,0 +1,189 @@
+---
+title: 'A guide to Windows development with Wasp'
```

Comment:
can we change the title to, "A Guide to Windows Development with Wasp & WSL"?

Reply:
Thanks for the advice. Done! 😄